apk info -q doesn't actually output anything, and also add 2>/dev/null to avoid
showing warnings like this when completing:
WARNING: opening /home/triallax/src/cports/packages/user: No such file or directory
diff --git a/share/completions/apk.fish b/share/completions/apk.fish
index 4d8fd6a9c..928e9956f 100644
--- a/share/completions/apk.fish
+++ b/share/completions/apk.fish
@@ -1,10 +1,10 @@
 # Completions for apk (Alpine Package Keeper)
 
 # Package name
-complete -c apk -n "__fish_seen_subcommand_from add" -a "(apk search -q)" -d Package
-complete -c apk -n "__fish_seen_subcommand_from manifest" -a "(apk info -q)" -d Package
-complete -f -c apk -n "__fish_seen_subcommand_from info fetch dot" -a "(apk search -q)" -d Package
-complete -f -c apk -n "__fish_seen_subcommand_from del fix version" -a "(apk info -q)" -d Package
+complete -c apk -n "__fish_seen_subcommand_from add" -a "(apk search -q 2>/dev/null)" -d Package
+complete -c apk -n "__fish_seen_subcommand_from manifest" -a "(apk info 2>/dev/null)" -d Package
+complete -f -c apk -n "__fish_seen_subcommand_from info fetch dot" -a "(apk search -q 2>/dev/null)" -d Package
+complete -f -c apk -n "__fish_seen_subcommand_from del fix version" -a "(apk info 2>/dev/null)" -d Package
 
 # Global options
 complete -f -c apk -s h -l help -d "Show help"
